dependencyManagement/build.gradle.kts

```diff
@@ -12,8 +12,8 @@ val dependencyVersions = hashMapOf<String, String>()
 rootProject.extra["versions"] = dependencyVersions
 
 val DEPENDENCY_BOMS = listOf(
-    "io.opentelemetry:opentelemetry-bom:1.0.0",
-    "io.opentelemetry:opentelemetry-bom-alpha:1.0.0-alpha"
+    "io.opentelemetry:opentelemetry-bom:1.3.0",
+    "io.opentelemetry:opentelemetry-bom-alpha:1.3.0-alpha"
```

Comment:
Mind updating the project version to 1.3.0-alpha-SNAPSHOT with these changes if you think it's the right time?